jest.config.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
 	preset: './gutenberg/node_modules/react-native/jest-preset.js',
 	setupFiles: [ '<rootDir>/' + configPath + '/setup.js', './jest.setup.js' ],
 	setupFilesAfterEnv: [ '<rootDir>/' + configPath + '/setup-after-env.js' ],
-	testMatch: [ '<rootDir>/src/**/test/*.[jt]s?(x)' ],
+	testMatch: [ '<rootDir>/src/**/test/**/*.[jt]s?(x)' ],
```

Comment:
In case want to add files with test helpers alongside the tests, this pattern would conflict. Not a blocker but we might adjust it in the future if we have this need.